packages/deepbook/sources/pool.move

```diff
@@ -985,13 +985,13 @@ public fun locked_balance<BaseAsset, QuoteAsset>(
         deep_quantity = deep_quantity + deep;
     });
 
-    let settled_balances = self
-        .state
-        .account(balance_manager.id())
-        .settled_balances();
-    base_quantity = base_quantity + settled_balances.base();
-    quote_quantity = quote_quantity + settled_balances.quote();
-    deep_quantity = deep_quantity + settled_balances.deep();
+    if (self.state.account_exists(balance_manager.id())) {
```

Comment:
Can we put this if statement in the beginning of the function, and keep the remaining code the same?

```
/// Returns the locked balance for the balance_manager in the pool.
/// Locked balance = open orders + settled balances.
/// Returns (base_quantity, quote_quantity, deep_quantity).
public fun locked_balance<BaseAsset, QuoteAsset>(
    self: &Pool<BaseAsset, QuoteAsset>,
    balance_manager: &BalanceManager,
): (u64, u64, u64) {
    let account_orders = self.get_account_order_details(balance_manager);
    let self = self.load_inner();
    if (!self.state.account_exists(balance_manager.id())) {
        return (0, 0, 0)
    };

    let mut base_quantity = 0;
    let mut quote_quantity = 0;
    let mut deep_quantity = 0;

    account_orders.do_ref!(|order| {
        let maker_fee = self.state.history().historic_maker_fee(order.epoch());
        let (base, quote, deep) = order.locked_balance(maker_fee);
        base_quantity = base_quantity + base;
        quote_quantity = quote_quantity + quote;
        deep_quantity = deep_quantity + deep;
    });

    let settled_balances = self
        .state
        .account(balance_manager.id())
        .settled_balances();
    base_quantity = base_quantity + settled_balances.base();
    quote_quantity = quote_quantity + settled_balances.quote();
    deep_quantity = deep_quantity + settled_balances.deep();

    (base_quantity, quote_quantity, deep_quantity)
}
```